### ISSUE TITLE
how to pass uniform parameter to shader?

### ISSUE THREAD

**Issue by mikeny07** [pygfx/pygfx#280]
The custom_object.py example shows how to create a custom object with the user supplied shader code. In that example,  it passes uniform buffer to shader at bindings0 with the pygfx-builtin  render_info.stdinfo_uniform. What's the template to define a custom uniform buffer and then pass it to shader? Does it need the device object in wgpu to create that buffer?

Thanks for the excellent work!

**Comment by almarklein** [comment]
Good point, that example was quite minimal. See #281 for more advanced usages.

**Comment by mikeny07** [comment]
Thank you for adding more examples for custom objects.
